includes/class-wc-ajax.php

```diff
@@ -861,31 +861,48 @@ public static function add_order_item() {
 			wp_die( -1 );
 		}
 
-		$response = array();
+		if ( ! isset( $_POST['order_id'] ) ) {
```

Comment:
Note to reviewer: I have refactored this method in two to add `maybe_add_order_item` so that I can unit test this better. Only real changes are deletion of these lines: https://github.com/woocommerce/woocommerce/pull/26642/files#diff-133db5662c51f5686d87611121a05a3bL918-L923. Other changes are rearranging of some lines to make unit testing easier.